examples/apollo/src/SolutionA/makeExecutableSchema.ts

```diff
@@ -27,6 +27,15 @@ export default function <TContext>(config: IExecutableSchemaDefinition<TContext>
         .join("\n");
 
     const typesSdl = gql(typesSdlRaw);
+    
+    // fetch OprExists as an Enum type
+    const oprExists = typesCache.OprExists as GraphQLEnumType;
```

Comment:
This is a problem for all enums, so maybe something along the lines of:
`Object.values(typesCache).filter(isEnumType).map(enumType => ...`